.github/workflows/publish_cli.yml

```diff
@@ -40,8 +40,9 @@ jobs:
       run: |
         cp ./README.md ./CLI-${{ matrix.os }}/
         cp ./LICENSE.txt ./CLI-${{ matrix.os }}/
+        cp ./.disable_gb1click ./CLI-${{ matrix.os }}/
     - name: Upload ${{ matrix.os }} CLI
       uses: actions/upload-artifact@v2
       with:
-        name: CLI-${{ matrix.os }}-isBundled-${{ matrix.bundled }}
-        path: CLI-${{ matrix.os }}
+        name: CLI-UTMTCE${{ matrix.os }}-isBundled-${{ matrix.bundled }}
+        path: CLI-UTMTCE${{ matrix.os }}
```

Comment:
You forgot about these changes (and in the other *.yml files)

Reply:
let me change that thing

Reply:
ok changed

Reply:
I see that you just addressed this comment, so you should mark this as resolved